packages/flutter_tools/lib/src/test/coverage_collector.dart

```diff
@@ -58,16 +58,18 @@ class CoverageCollector {
   List<Future<Null>> _jobs = <Future<Null>>[];
   Map<String, dynamic> _globalHitmap;
 
-  Future<String> finalizeCoverage() async {
+  Future<String> finalizeCoverage({Formatter formatter}) async {
```

Comment:
And in my case, apparently illiterate! Missed this comment; will send a quick patch :)